tritonbench/operators/flash_attention/operator.py

```diff
@@ -136,6 +136,18 @@
 except (ImportError, IOError, AttributeError, TypeError):
     HAS_XFORMERS = False
 
+# [Optional] TLX backend
+try:
```

Comment:
We introduced a cleaner version of the guards in https://github.com/meta-pytorch/tritonbench/pull/387. This can be simplified to 
```suggestion
from tritonbench.utils.python_utils import try_import
with try_import("HAS_TLX"):
    import triton.language.extra.tlx as tlx
```